### PR TITLE
[EG-2120] Minor typos on some pages

### PR DESCRIPTION
**Pull request by maryforde**
Fixed typo "results your were expecting" across all versions.

Also fixed typo "lover-level certificates" across the following versions:

- CENM 1.0
- CENM 1.1
- Corda Enterprise 4.4

### CODE REVIEW

File: content/en/docs/corda-enterprise/3.0/api-vault-query.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ There are four implementations of this interface which can be chained together t
 
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
-CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).> 
+CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.0/api-vault-query.md:163]: Remove the closing angle bracket - I guess this was left from the `rst` formatting when the docs were migrated

File: content/en/docs/corda-enterprise/3.0/api-vault-query.md [hunk 2]
```diff
@@ -169,7 +169,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.0/api-vault-query.md:172]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.0/api-vault-query.md [hunk 3]
```diff
@@ -178,7 +178,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.0/api-vault-query.md:181]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.0/api-vault-query.md [hunk 4]
```diff
@@ -192,7 +192,7 @@ documentation and associated examples. Custom criteria expressions are expressed
 types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg,
 max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple
 construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in
-`QueryCriteriaUtils` for a complete specification of the DSL.> 
+`QueryCriteriaUtils` for a complete specification of the DSL.>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.0/api-vault-query.md:195]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.1/api-vault-query.md [hunk 5]
```diff
@@ -160,7 +160,7 @@ There are four implementations of this interface which can be chained together t
 
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
-CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).> 
+CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.1/api-vault-query.md:163]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.1/api-vault-query.md [hunk 6]
```diff
@@ -169,7 +169,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.1/api-vault-query.md:172]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.1/api-vault-query.md [hunk 7]
```diff
@@ -178,7 +178,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.1/api-vault-query.md:181]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.1/api-vault-query.md [hunk 8]
```diff
@@ -192,7 +192,7 @@ documentation and associated examples. Custom criteria expressions are expressed
 types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg,
 max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple
 construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in
-`QueryCriteriaUtils` for a complete specification of the DSL.> 
+`QueryCriteriaUtils` for a complete specification of the DSL.>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.1/api-vault-query.md:195]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.2/api-vault-query.md [hunk 9]
```diff
@@ -160,7 +160,7 @@ There are four implementations of this interface which can be chained together t
 
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
-CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).> 
+CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.2/api-vault-query.md:163]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.2/api-vault-query.md [hunk 10]
```diff
@@ -169,7 +169,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.2/api-vault-query.md:172]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.2/api-vault-query.md [hunk 11]
```diff
@@ -178,7 +178,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.2/api-vault-query.md:181]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.2/api-vault-query.md [hunk 12]
```diff
@@ -192,7 +192,7 @@ documentation and associated examples. Custom criteria expressions are expressed
 types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg,
 max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple
 construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in
-`QueryCriteriaUtils` for a complete specification of the DSL.> 
+`QueryCriteriaUtils` for a complete specification of the DSL.>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.2/api-vault-query.md:195]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.3/api-vault-query.md [hunk 13]
```diff
@@ -166,7 +166,7 @@ There are four implementations of this interface which can be chained together t
 
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
-CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).> 
+CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.3/api-vault-query.md:169]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.3/api-vault-query.md [hunk 14]
```diff
@@ -175,7 +175,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.3/api-vault-query.md:178]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.3/api-vault-query.md [hunk 15]
```diff
@@ -184,7 +184,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.3/api-vault-query.md:187]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/3.3/api-vault-query.md [hunk 16]
```diff
@@ -198,7 +198,7 @@ documentation and associated examples. Custom criteria expressions are expressed
 types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg,
 max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple
 construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in
-`QueryCriteriaUtils` for a complete specification of the DSL.> 
+`QueryCriteriaUtils` for a complete specification of the DSL.>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/3.3/api-vault-query.md:201]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.0/api-vault-query.md [hunk 17]
```diff
@@ -173,7 +173,7 @@ There are four implementations of this interface which can be chained together t
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
 CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED),
-state constraints (see [Constraint Types](api-contract-constraints.md#implicit-constraint-types)), relevancy (ALL, RELEVANT, NON_RELEVANT).> 
+state constraints (see [Constraint Types](api-contract-constraints.md#implicit-constraint-types)), relevancy (ALL, RELEVANT, NON_RELEVANT).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.0/api-vault-query.md:176]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.0/api-vault-query.md [hunk 18]
```diff
@@ -182,7 +182,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.0/api-vault-query.md:185]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.0/api-vault-query.md [hunk 19]
```diff
@@ -191,7 +191,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.0/api-vault-query.md:194]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.0/api-vault-query.md [hunk 20]
```diff
@@ -205,7 +205,7 @@ documentation and associated examples. Custom criteria expressions are expressed
 types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg,
 max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple
 construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in
-`QueryCriteriaUtils` for a complete specification of the DSL.> 
+`QueryCriteriaUtils` for a complete specification of the DSL.>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.0/api-vault-query.md:208]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.1/api-vault-query.md [hunk 21]
```diff
@@ -173,7 +173,7 @@ There are four implementations of this interface which can be chained together t
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
 CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED),
-state constraints (see [Constraint Types](api-contract-constraints.md#implicit-constraint-types)), relevancy (ALL, RELEVANT, NON_RELEVANT).> 
+state constraints (see [Constraint Types](api-contract-constraints.md#implicit-constraint-types)), relevancy (ALL, RELEVANT, NON_RELEVANT).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.1/api-vault-query.md:176]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.1/api-vault-query.md [hunk 22]
```diff
@@ -182,7 +182,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.1/api-vault-query.md:185]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.1/api-vault-query.md [hunk 23]
```diff
@@ -191,7 +191,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.1/api-vault-query.md:194]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.1/api-vault-query.md [hunk 24]
```diff
@@ -205,7 +205,7 @@ documentation and associated examples. Custom criteria expressions are expressed
 types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg,
 max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple
 construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in
-`QueryCriteriaUtils` for a complete specification of the DSL.> 
+`QueryCriteriaUtils` for a complete specification of the DSL.>
 {{< note >}}
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.1/api-vault-query.md:209]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.2/api-vault-query.md [hunk 25]
```diff
@@ -173,7 +173,7 @@ There are four implementations of this interface which can be chained together t
 
 * `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED,
 CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED),
-state constraints (see [Constraint Types](api-contract-constraints.md#implicit-constraint-types)), relevancy (ALL, RELEVANT, NON_RELEVANT).> 
+state constraints (see [Constraint Types](api-contract-constraints.md#implicit-constraint-types)), relevancy (ALL, RELEVANT, NON_RELEVANT).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.2/api-vault-query.md:176]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.2/api-vault-query.md [hunk 26]
```diff
@@ -182,7 +182,7 @@ locked states).{{< /note >}}
 * `FungibleAssetQueryCriteria` provides filterable criteria on attributes defined in the Corda Core
 `FungibleAsset` contract state interface, used to represent assets that are fungible, countable and issued by a
 specific party (eg. `Cash.State` and `CommodityContract.State` in the Corda finance module). Filterable
-attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).> 
+attributes include: participants(s), owner(s), quantity, issuer party(s) and issuer reference(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.2/api-vault-query.md:185]: Remove the closing angle bracket

File: content/en/docs/corda-enterprise/4.2/api-vault-query.md [hunk 27]
```diff
@@ -191,7 +191,7 @@ state attributes to the **vault_fungible_states** table.{{< /note >}}
 * `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState`
 and `DealState` contract state interfaces, used to represent entities that continuously supersede themselves, all
 of which share the same `linearId` (e.g. trade entity states such as the `IRSState` defined in the SIMM
-valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.2/api-vault-query.md:194]: Remove the closing angle bracket - apply for all other files/places

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.2/api-vault-query.md:194]: Also, some of the attributes should perhaps be in code style e.g. `participant`(s), `linearId`(s), `uuid`(s), and `externalId`(s) - although the (s) at the end doesn't read very well, I think you can remove that part and rephrase to something like "Filterable attributes include one or more of the following: `participant`, `linearId`, ..."

File: content/en/docs/corda-os/1.0/api-vault-query.md [hunk 28]
```diff
@@ -157,22 +157,22 @@ The `QueryCriteria` interface provides a flexible mechanism for specifying diffe
 There are four implementations of this interface which can be chained together to define advanced filters.
 
 
-* `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED, CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).> 
+* `VaultQueryCriteria` provides filterable criteria on attributes within the Vault states table: status (UNCONSUMED, CONSUMED), state reference(s), contract state type(s), notaries, soft locked states, timestamps (RECORDED, CONSUMED).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/1.0/api-vault-query.md:160]: values should also be in code style - (`UNCONSUMED`, `CONSUMED`) - across all docs

Comment (ivanterziev-r3) [reply content/en/docs/corda-os/1.0/api-vault-query.md:160]: P.S. I know this is not in the scope of this task but a good opportunity to fix some more issues in the content.

File: content/en/docs/corda-os/1.0/api-vault-query.md [hunk 29]
```diff
@@ -157,22 +157,22 @@ The `QueryCriteria` interface provides a flexible mechanism for specifying diffe
...
 {{< note >}}
 All contract states that extend the `FungibleAsset` now automatically persist that interfaces common state attributes to the **vault_fungible_states** table.{{< /note >}}
 
 
-* `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState` and `DealState` contract state interfaces, used to represent entities that continuously supercede themselves, all of which share the same *linearId* (eg. trade entity states such as the `IRSState` defined in the SIMM valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).> 
+* `LinearStateQueryCriteria` provides filterable criteria on attributes defined in the Corda Core `LinearState` and `DealState` contract state interfaces, used to represent entities that continuously supercede themselves, all of which share the same *linearId* (eg. trade entity states such as the `IRSState` defined in the SIMM valuation demo). Filterable attributes include: participant(s), linearId(s), uuid(s), and externalId(s).>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/1.0/api-vault-query.md:170]: "eg." is misspelled, and in general I would avoid using e.g., i.e., and so on:

(for example, trade entity states such as the `IRSState` defined in the SIMM valuation demo)

File: content/en/docs/corda-os/1.0/api-vault-query.md [hunk 30]
```diff
@@ -157,22 +157,22 @@ The `QueryCriteria` interface provides a flexible mechanism for specifying diffe
...
+* `VaultCustomQueryCriteria` provides the means to specify one or many arbitrary expressions on attributes defined by a custom contract state that implements its own schema as described in the [Persistence](api-persistence.md) documentation and associated examples. Custom criteria expressions are expressed using one of several type-safe `CriteriaExpression`: BinaryLogical, Not, ColumnPredicateExpression, AggregateFunctionExpression. The `ColumnPredicateExpression` allows for specification arbitrary criteria using the previously enumerated operator types. The `AggregateFunctionExpression` allows for the specification of an aggregate function type (sum, avg, max, min, count) with optional grouping and sorting. Furthermore, a rich DSL is provided to enable simple construction of custom criteria using any combination of `ColumnPredicate`. See the `Builder` object in `QueryCriteriaUtils` for a complete specification of the DSL.>
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/1.0/api-vault-query.md:175]: values in code style

File: content/en/docs/corda-os/1.0/api-vault-query.md (hunk 28 above)
Comment (maryforde) [reply content/en/docs/corda-os/1.0/api-vault-query.md:160]: As discussed, I've actually removed the formatting on all of these queries again, as the attribute names are quite different to the 'layperson language' used here.

**Pull request merged**